src/components/a-input/_preview.html

```diff
@@ -2,11 +2,12 @@
 <axa-input value="Pre-filled content" name="test-name" inline></axa-input>
 <axa-input value="Pre-filled content" name="test-name" inline></axa-input>
 <axa-input value="Pre-filled content" name="test-name" inline></axa-input>
-<axa-input value="Pre-filled content" name="test-name" disabled></axa-input>
-<axa-input value="Pre-filled content" name="test-name" error></axa-input>
+<axa-input value="Pre-filled content - Disabled" name="test-name" disabled></axa-input>
+<axa-input value="Pre-filled content - Error" name="test-name" error></axa-input>
 <axa-input placeholder="Pre-filled content" name="test-name"></axa-input>
 <axa-input value="Pre-filled content" name="test-name" valid></axa-input>
 <axa-input value="Pre-filled content" name="test-name" valid inline></axa-input>
 <axa-input value="Pre-filled content" name="test-name" valid inline></axa-input>
 <axa-input value="Pre-filled content" name="test-name" valid inline></axa-input>
-<axa-input placeholder="Disabled paste" disable-paste name="test-name"></axa-input>
+<axa-input placeholder="Pre-filled content - Disabled paste" disable-paste name="test-name"></axa-input>
```

Comment:
@nakovskijosif 
@LucaMele 
I wonder what `disable-paste` is supposed to be, it is only defined as boolean proptype `disablePaste` below, but without any usage 🤔 

If you agree I would suggest to remove it.

Reply:
This is a feature I created a while ago, disables paste on the input field. I only modified the preview option a bit on this PR.

Reply:
Ah I see, thanks for clarifying:)

I found the implementation here:
https://github.com/axa-ch/patterns-library/blob/cc8a91c3b4fd63d757de051a04651b0169fa8749/src/components/a-input/js/input.js#L14-L40